src/TokenHandler/AccountManager.mo

```diff
@@ -102,6 +150,16 @@ module {
       );
       log(ownPrincipal, #feeUpdated({ old = fee_; new = newFee }));
       fee_ := newFee;
+      // check if deposit minimum is updated
+      let newDepositMin = depositMinimum();
+      if (prevDepositMin != newDepositMin) {
+        log(ownPrincipal, #depositMinimumUpdated({ old = prevDepositMin; new = newDepositMin }));
+      };
```

Comment:
Maybe these lines can be in a function because they are currently called 4 times.

Reply:
Within updateFee() it's a bit tricky to reuse the logic for different minimums because we need to take into account the previous minimum.